buildtest/defaults.py

```diff
@@ -31,11 +31,12 @@
 
 # dictionary used for storing status of builds
 BUILDTEST_BUILD_HISTORY = {}
-BUILDTEST_CONFIG_FILE = os.path.join(BUILDTEST_ROOT, "settings.yml")
-BUILDTEST_CONFIG_BACKUP_FILE = os.path.join(BUILDTEST_ROOT, "settings.yml.bak")
+BUILDTEST_CONFIG_FILE = os.path.join(BUILDTEST_ROOT, "settings.json")
```

Comment:
Why did you change the settings file to be json instead? Why should we not be consistent?

Reply:
I found it easier to write configuration in json. It was a personal choice and buildtest setting does not have to be bound to YAML.

Reply:
I can't argue with that, although it does seem inconsistent.